packages/template-cli/templates/src/ssr/.npmrc

```diff
@@ -0,0 +1 @@
+//registry.npmjs.org/:_authToken=${NPM_TOKEN}
```

Comment:
We have to write some documentation around this right?

Reply:
This is tech debt and needs to be removed once the other pipeline changes are merged. https://github.com/amido/stacks-pipeline-templates/pull/18